snowpack/src/commands/dev.ts

```diff
@@ -138,7 +138,7 @@ const sendFile = (
     stream.pipeline(bodyStream, zlib.createGzip(), res, function onError(err) {
       if (err) {
         res.end();
-        logger.error(`✘ An error occurred serving ${colors.bold(req.url)}`);
+        logger.error(`✘ An error occurred serving ${colors.bold(req.url!)}`);
```

Comment:
colors.bold no longer takes undefined, which `req.url` is technically typed as `string | undefined` from Node (but in practice, is always string)